src/Compilers/CSharp/Desktop/CSharpCodeAnalysis.Desktop.csproj

```diff
@@ -38,7 +38,6 @@
     </ProjectReference>
   </ItemGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DefineConstants>TRACE;DEBUG</DefineConstants>
```

Comment:
@tmeschter they are already present in Debug builds, these definitions were extraneous.